server.js

```diff
@@ -9,8 +9,8 @@ var express = require('express'),
 
 // Prefix paths with tool name
 var basedir = '/hatjitsu';
-// Use port provided by portgrabber
-var port = process.env.PORT;
+// Use port provided by portgrabber or default to 5000
+var port = process.env.PORT || 5000;
```

Comment:
The port grabber magic only happens on Toolforge as part of the `webservice start` command's activity, so if you were trying to test things locally this default port is a good idea.